src/main/java/uk/nhs/digital/nhsconnect/lab/results/utils/UUIDGenerator.java

```diff
@@ -0,0 +1,12 @@
+package uk.nhs.digital.nhsconnect.lab.results.utils;
+
+import java.util.UUID;
+
+public final class UUIDGenerator {
+
+    private UUIDGenerator() { }
+
+    public static String generateUUID() {
+        return UUID.randomUUID().toString();
+    }
+}
```

Comment:
If this is made a Singleton then it can be injected and mocked for tests.
```suggestion
@Component
public class UUIDGenerator {
    public String generateUUID() {
        return UUID.randomUUID().toString();
    }
}
```